### PR TITLE
Add security.md

### PR DESCRIPTION
**Pull request by lumjjb**
Adding security.md and list of maintainers to contact. 

@swinslow @RishabhBhatnagar can you help verify your information in the file: MAINTAINERS.

Signed-off-by: Brandon Lum <lumjjb@gmail.com>

### CODE REVIEW

**Comment by RishabhBhatnagar** [comment]
This LGTM, @swinslow, I was wondering if we should also point out to the gitter channel if it's our official point of contact.

**Review by swinslow — APPROVED** [review]

**Comment by swinslow** [comment]
LGTM, thanks!

@RishabhBhatnagar I'll defer to you and @lumjjb, but I rarely use the Gitter channel so I wouldn't be inclined to list it as an official channel.

**Review by ded1976 — APPROVED** [review]

**Pull request merged**
